alternator/executor.cc

```diff
@@ -200,19 +200,23 @@ executor::executor(gms::gossiper& gossiper,
          db::system_distributed_keyspace& sdks,
          cdc::metadata& cdc_metadata,
          smp_service_group ssg,
-         utils::updateable_value<uint32_t> default_timeout_in_ms)
+         utils::updateable_value<uint32_t> default_timeout_in_ms,
+         utils::updateable_value<uint32_t> expression_max_cache_size)
     : _gossiper(gossiper),
       _proxy(proxy),
       _mm(mm),
       _sdks(sdks),
       _cdc_metadata(cdc_metadata),
       _enforce_authorization(_proxy.data_dictionary().get_config().alternator_enforce_authorization()),
-      _ssg(ssg)
+      _ssg(ssg),
+      _expression_factory(std::make_unique<expression_factory>(expression_factory::config{std::move(expression_max_cache_size)}, _stats))
```

Comment:
Why does _expression_factory need to be a unique_ptr and can't be a direct member?
(maybe I'll see the answer later, this is a big patch...)